python/ray/data/_internal/execution/autoscaler/__init__.py

```diff
@@ -0,0 +1,15 @@
+from .autoscaler import Autoscaler
+from .autoscaling_actor_pool import AutoscalingActorPool
+from .default_autoscaler import DefaultAutoscaler
+
+
+def create_autoscaler(topology, resource_manager, execution_id):
+    return DefaultAutoscaler(topology, resource_manager, execution_id)
```

Comment:
What's the motivation behind adding this function? Seems like this is a pass-through method right now, and I'm wondering if it'd make sense to just directly constructor the `Autoscaler`.

Reply:
The purpose is to make it easier to support pluggability. Later we can choose different Autoscaler implementations here.